parl/utils/communication.py

```diff
@@ -115,3 +120,23 @@ def loads_return(data):
         raise DeserializeError(e)
 
     return ret
+
+
+#Reference: https://stackoverflow.com/questions/2953462/pinging-servers-in-python
+def ping(host):
+    """
+    Returns True if host (str) responds to a ping request.
+    Remember that a host may not respond to a ping (ICMP) request even if the host name is valid.
+    """
+
+    # Option for the number of packets as a function of
+    param = '-n' if platform.system().lower() == 'windows' else '-c'
```

Comment:
We have `_IS_WINDOWS ` in https://github.com/PaddlePaddle/PARL/blob/c24e283c735630ab7995ac3556392b8cfa131736/parl/utils/utils.py#L100

Reply:
fixed